src/v/archival/ntp_archiver_service.cc

```diff
@@ -422,8 +422,9 @@ ss::future<> ntp_archiver::upload_topic_manifest() {
         } else {
             _topic_manifest_dirty = false;
         }
-    } catch (const ss::gate_closed_exception& err) {
-    } catch (const ss::abort_requested_exception& err) {
+    } catch (const ss::gate_closed_exception&) {
+    } catch (const ss::broken_named_semaphore&) {
```

Comment:
nit: is this something we need to handle at every transitive caller to `client_pool.acquire()`? Might be worth considering handling this down deeper in the client pool, given there is also some handling for brokern condition variables there?

Reply:
> is this something we need to handle at every transitive caller to client_pool.acquire()?

Yes.

> Might be worth considering handling this down deeper in the client pool, given there is also some handling for brokern condition variables there?

Agreed, our approach for handling shutdown errors of the client pool isn't very nice. We just catch every possible one when when we make a call into the remote. We should tidy this up a bit, but I'd say let's do it
in a separate PR.